proto/panacea/oracle/v2/oracle.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+package panacea.oracle.v2;
+
+option go_package = "github.com/medibloc/panacea-core/x/oracle/types";
+
+import "gogoproto/gogo.proto";
+
+// Oracle defines a detail of oracle.
+message Oracle {
+  string operator_address = 1;
+  OracleStatus status = 2;
```

Comment:
I think this status is not needed. 
If it is for upgrade, we can use `unique_id` to know whether the oracles have been upgraded

Reply:
You're right.
I think currently not required.
I will remove soon.

Reply:
fixed: https://github.com/medibloc/panacea-core/pull/515/commits/8eccb41513d2d18b87ff343c73155def0844c388